pkg/ee/metering/cronjob.go

```diff
@@ -115,7 +115,7 @@ mc mirror --newer-than "65d0h0m" s3/$S3_BUCKET /metering-data || true`,
 			job.Spec.JobTemplate.Spec.Template.Spec.Containers = []corev1.Container{
 				{
 					Name:  "kubermatic-metering-report",
-					Image: "quay.io/kubermatic/metering:7a12117",
+					Image: "quay.io/kubermatic/metering:v0.5",
```

Comment:
NIT. would be nice to set `ImagePullPolicy: corev1.PullAlways`

Reply:
Added in 92f17f1dc5bc4f0cc3e7fc621a442614e5553a73 :+1: